lib/logger/easy_logging_starter.js

```diff
@@ -21,10 +22,13 @@ let initTrialParameters;
 exports.init = async function (configFilePathFromConnectionString) {
   try {
     if (!allowedToInitialize(configFilePathFromConnectionString)) {
+      Logger.getInstance().info('Skipping Easy Logging initialization as it is not allowed to initialize');
       return;
     }
+    Logger.getInstance().info('Trying to initialize Easy Logging');
```

Comment:
it looks like debug or even trace log - please revalidate levels of messages in this PR

Reply:
This is on par with the `gosnowflake` logging levels. We had a meeting with ProdSec team and we should log the information about the tries.